rama-cluster/cluster.tf

```diff
@@ -41,7 +41,7 @@ variable "supervisor_volume_size_gb" {
 
 variable "use_private_ip" {
   type    = bool
-  default = false
+  default = true
```

Comment:
I don't think we should change the default for this. It's outside the scope of this issue.